src/stories/components/ChecDropdown.stories.mdx

```diff
@@ -371,3 +371,62 @@ Todo list
     }}
   </Story>
 </Preview>
+
+# With long labels
```

Comment:
These shouldn't be h1s, minor point

Reply:
Yeah I thought this, but I was just copying the other titles.